### DISCUSSION TITLE
Container health portainer

### DISCUSSION THREAD

**Discussion by ppeterrr** [qdm12/gluetun#966] [Ideas]
Sorry for this again. But i updated and now i notice this error in portainer.

While everything seems to run fine. I dont know how to solve this.

![Schermafbeelding 2022-04-27 125327](https://user-images.githubusercontent.com/96293249/165503342-a7cac428-b751-43ee-bb67-e953fae418b2.png)

**Comment by qdm12** [comment]
Remove Gluetun from Portainer and re-add it again. It's a Portainer bug where the healthcheck command doesn't get updated with the newer image. The healthcheck command did change maybe one or two months ago.

**Reply by ppeterrr** [reply]
Thanks for your swift reply, i will do that soon. Good to know that it aint a problem with Gluetun itself.
